script/strip-binaries.py

```diff
@@ -22,7 +22,7 @@ def strip_binary(binary_path, target_cpu):
     strip = 'mips64el-redhat-linux-strip'
   else:
     strip = 'strip'
-  execute([strip, '--preserve-dates', binary_path])
+  execute([strip, '-x', '-S', '--preserve-dates', binary_path])
```

Comment:
Prefer long-style switches in scripts, as they self-document to some extent.

Also, from the man-page, `-x` means "remove all non-global symbols", and `-S` means "remove debugging symbols only". How can these not conflict?

```suggestion
  execute([strip, '--discard-all', '--strip-debug', '--preserve-dates', binary_path])
```

Reply:
These are the two flags included by default in chromium macOS. Isn't possible to have global debugging symbols?

Reply:
Hm, I'm not sure. If Chromium combines them that's a reasonable hint that it might be useful to do so.

Reply:
Just to be noted that `--discard-all` and `--strip-debug` does not seem to be POSIX compatible flags. Most probably they are only available on GNU strip. Do we still want to use them verbatim?

Reply:
This script is Linux-only.